Coursework/1-If-Else/exercise5.js

```diff
@@ -1,13 +1,22 @@
 // I would like to buy a sandwich. Do I have enough money?
 
-let myMoney = 1.66;
+let myMoney = 3.3;
 
-let sandwichCost = 2.50;
+let sandwichCost = 2.5;
 
 // Write an "if" statement below to print either - 
 // "You can afford the sandwich :-)" or
 // "You don't have enough money :-("
+if(myMoney >= sandwichCost){
+   difference = myMoney - sandwichCost;
+   change = Math.floor(difference*100)/100;

```

Comment:
hey Megumi, 
just for your information, you can check this way too for working around strange JS behaviour with decimal numbers:
```javascript
(3.3-2.5).toFixed(2);
```
`toFixed()` method comes from [Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/number): 
   [Number.prototype.toFixed()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed)